### PR TITLE
Added numerical.divide() and numerical.multiply() methods

### PR DESCRIPTION
**Pull request by bharadwajyarlagadda**

### CODE REVIEW

**Comment by bharadwajyarlagadda** [comment]
@dgilland

File: pydash/numerical.py [hunk 1]
```diff
@@ -117,6 +119,29 @@ def average(collection, callback=None):
 mean = average
 
 
+def call_math_operator(value1, value2, op, default):
```
Comment (dgilland) [inline pydash/numerical.py]: Generally, the helper functions are defined alllll the way at the bottom: https://github.com/dgilland/pydash/blob/8ca3261ba7a8a28c681268df48d23b399774312b/pydash/numerical.py#L515

So if you're moving the function now, best to put it there.

**Comment by dgilland** [comment]
LGTM after moving the helper function to bottom of module and squashing :+1:

**Comment by bharadwajyarlagadda** [comment]
@dgilland

**Pull request merged**